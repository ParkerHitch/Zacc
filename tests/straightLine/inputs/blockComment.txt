
// Very important text here
print(1234.5678);


/* And some much less important stuff here:

a0 <- 1;

blahblahblah
lorem ipsem
// woooooo * sjdlf */

print(45.67);
