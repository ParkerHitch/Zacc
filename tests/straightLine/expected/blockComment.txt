PRINT: 1234.5678
PRINT: 45.6700
